gnocchi/rest/aggregates/operations.py

```diff
@@ -183,20 +181,22 @@ def handle_resample(agg, granularity, timestamps, values, is_aggregated,
             new_values = numpy.array([ts["values"]])
         else:
             new_values = numpy.append(new_values, [ts["values"]], axis=0)
-    return sampling, result_timestamps, new_values.T, is_aggregated
+    return sampling, result_timestamps, new_values.T
 
 
 def handle_aggregation_operator(nodes, granularity, timestamps, initial_values,
-                                is_aggregated, references):
+                                references):
     op = aggregation_operators[nodes[0]]
     agg = nodes[1]
     subnodes = nodes[-1]
     args = nodes[2:-1]
     granularity, timestamps, values, is_aggregated = evaluate(
         subnodes, granularity, timestamps, initial_values,
-        is_aggregated, references)
-    return op(agg, granularity, timestamps, values, is_aggregated,
-              references, *args)
+        references)
+    if op == handle_aggregate:
+        is_aggregated = True
```

Comment:
I preferred the version where each methods return False or True.

Reply:
Do you mean that the `handle_*` functions still has to be called with `is_aggregated`? I modified this part since only `handle_aggregate` returns `True` all the time and the others just return back `is_aggregated` without any modification.

Reply:
yeah, i'm not sure this patch cleans up the interface any better. it's probably more confusing  because there's just random false/true parameters passed around.

we could remove it completely if 'operations' was a string... that way we just have to check if there's an aggregate operation or if the number of result series < number of input series.